File: 프로그래머스/0/120891. 369게임/369게임.cs
using System;

public class Solution 
{
    public int solution(int order) 
    {
        string orderStr = order.ToString();
        
        int count = 0;
        for(int i=0; i<orderStr.Length; i++) {
            switch(orderStr[i].ToString()) {
                case "3":
                case "6":
                case "9":
                    count++;
                    break;
                default:
                    break;     
            }
        }
        return count;
        
    }
}

File: 프로그래머스/0/120891. 369게임/369게임.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int order) 
{
    int count = 0;
    for(char &c : to_string(order))
    {
        if((c - '0') != 0 && (c - '0')%3 == 0)
        {
            count++;
        }
    }
    return count;
}